### PR TITLE
(FM-8073) Support blocks in apply_manifest

### PR DESCRIPTION
**Pull request by tphoney**

### CODE REVIEW

File: lib/puppet_litmus/serverspec.rb [hunk 1]
```diff
@@ -22,6 +22,9 @@ def idempotent_apply(manifest)
   #  :catch_changes [Boolean] exit status of 1 if there were changes.  
   #  :expect_failures [Boolean] doesnt return an exit code of non-zero if the apply failed.  
   #  :manifest_file_location [Path] The place on the target system.
+  #  :prefix_command [String] prefixes the puppet apply command; eg "export LANGUAGE='ja';".
```
Comment (ekohl) [inline lib/puppet_litmus/serverspec.rb]: The semi colon is not needed:
```suggestion
  #  :prefix_command [String] prefixes the puppet apply command; eg "export LANGUAGE='ja'".
```

File: lib/puppet_litmus/rake_tasks.rb [hunk 2]
```diff
@@ -269,23 +271,27 @@ def run_local_command(command)
...
         results.each do |result|
-          failures = true unless result.last.exitstatus.zero?
           puts result
         end
-        exit 1 if failures
+
+        # output test summary
+        puts "Successful on #{success_list.size} nodes: #{success_list}" unless success_list.size.zero?
+        puts "Failed on #{failure_list.size} nodes: #{failure_list}" unless failure_list.size.zero?
+        exit 1 unless failure_list.size.zero?
```
Comment (ekohl) [inline lib/puppet_litmus/rake_tasks.rb]:
```suggestion
        exit 1 if failure_list.any?
```

File: lib/puppet_litmus/rake_tasks.rb [hunk 3]
```diff
@@ -269,23 +271,27 @@ def run_local_command(command)
...
-        failures = false
+        # output test results
         results.each do |result|
-          failures = true unless result.last.exitstatus.zero?
           puts result
         end
-        exit 1 if failures
+
+        # output test summary
+        puts "Successful on #{success_list.size} nodes: #{success_list}" unless success_list.size.zero?
```
Comment (ekohl) [inline lib/puppet_litmus/rake_tasks.rb]:
```suggestion
        puts "Successful on #{success_list.size} nodes: #{success_list}" if success_list.any?
```

File: lib/puppet_litmus/rake_tasks.rb [hunk 4]
```diff
@@ -269,23 +271,27 @@ def run_local_command(command)
...
+        puts "Failed on #{failure_list.size} nodes: #{failure_list}" unless failure_list.size.zero?
```
Comment (ekohl) [inline lib/puppet_litmus/rake_tasks.rb]:
```suggestion
        puts "Failed on #{failure_list.size} nodes: #{failure_list}" if failure_list.any?
```

**Comment by tphoney** [comment]
many thanks @ekohl !

**Comment by florindragos** [comment]
👍

**Pull request merged**